docs/setup/kubeedge_install_source.md

```diff
@@ -0,0 +1,165 @@
+
++ (**Optional**) Run `admission`, this feature is still being evaluated.
+    please read the docs in [install the admission webhook](../../build/admission/README.md)
+
+## Setup Edge Node (KubeEdge Worker Node)
+
+1. Clone KubeEdge
+
+    + Setup [$GOPATH ](https://github.com/golang/go/wiki/SettingGOPATH) to clone the KubeEdge repository in the `$GOPATH`.
+
+        ```shell
+        git clone https://github.com/kubeedge/kubeedge.git $GOPATH/src/github.com/kubeedge/kubeedge
+        cd $GOPATH/src/github.com/kubeedge/kubeedge
+        ```
+
+2. Compile Edgecore
+
+    ```shell
+    cd $GOPATH/src/github.com/kubeedge/kubeedge
+    make all WHAT=edgecore
+    ```
+
+    KubeEdge can also be cross compiled to run on ARM based processors.
+    Please follow the instructions given below or click [Cross Compilation](cross-compilation.md) for detailed instructions.
+
+    ```shell
+    cd $GOPATH/src/github.com/kubeedge/kubeedge/edge
+    make cross_build
+    ```
+
+    KubeEdge can also be compiled with a small binary size. Please follow the below steps to build a binary of lesser size:
+
+    ```shell
+    apt-get install upx-ucl
+    cd $GOPATH/src/github.com/kubeedge/kubeedge/edge
+    make edge_small_build
+    ```
+
+    **Note:** If you are using the smaller version of the binary, it is compressed using upx, therefore the possible side effects of using upx compressed binaries like more RAM usage,
+    lower performance, whole code of program being loaded instead of it being on-demand, not allowing sharing of memory which may cause the code to be loaded to memory
+    more than once etc. are applicable here as well.
+
+    **Note:** If you don't want to compile, you may perform the next step
+
+    + Download KubeEdge from [Releases](https://github.com/kubeedge/kubeedge/releases)   
+    
+        Download `kubeedge-$VERSION-$OS-$ARCH.tar.gz` from above link. It would contain Edgecore and the configuration files.
+
+3. Copy edgecore binary/ Setup as a systemctl
+
+ + Run edgecore with systemd
+
+    It is also possible to start the edgecore with systemd. If you want, you could use the example systemd-unit-file. 
+
+    ```shell
+    sudo ln build/tools/edgecore.service /etc/systemd/system/edgecore.service
+    sudo systemctl daemon-reload
+    sudo systemctl start edgecore
+    ```
+    **Note:** Please fix __ExecStart__ path in edgecore.service. Do __NOT__ use relative path, use absoulte path instead.
+
+    If you also want also an autostart, you have to execute this, too:
+
+    ```shell
+    sudo systemctl enable edgecore
+    ```
+
+ + Copy edgecore file and sample config in a new directory
+
+    ```
+    cp $GOPATH/src/github.com/kubeedge/kubeedge/edge/edgecore ~/cmd/
+    cd ~/cmd
+    ```
```

Comment:
- missing path creation (I think we didn't need it here)
- missing copy of the configuration
- missing start command from shell